odd-platform-api/src/main/java/com/provectus/oddplatform/dto/DataEntityIngestionDtoSplit.java

```diff
@@ -18,6 +18,10 @@
         return Stream.concat(newEntities.stream(), existingEntities.stream()).collect(Collectors.toList());
     }
 
+    public List<Long> getIds() {
+        return this.getAllEntities().stream().map(EnrichedDataEntityIngestionDto::getId).collect(Collectors.toList());
```

Comment:
Won't there be a new `List` instance created on each method call?

Reply:
Fixed